webapp/src/client/client.js

```diff
@@ -271,29 +257,62 @@ export default class Client {
             automaticLeaderAssignment: 'SameEnterprise',
         };
 
-        const response = await request.
-            post(url).
-            set('Authorization', 'Bearer ' + appAccessToken).
-            set('Accept', 'application/json').
-            send(data);
+        const response = await this.doPost(url, data, {Authorization: 'Bearer ' + appAccessToken, Accept: 'application/json'});
 
         return {
-            meetingId: response.body.onlineMeetingId,
-            meetingUrl: response.body.joinUrl,
+            meetingId: response.onlineMeetingId,
+            meetingUrl: response.joinUrl,
         };
     };
 
-    sendPost = async (url, body, headers = {}) => {
-        headers['X-Requested-With'] = 'XMLHttpRequest';
+    doGet = async (url, headers = {}, credentials) => {
+        const options = {
+            method: 'get',
+            headers,
+        };
+
+        const newOptions = Client4.getOptions(options);
+
+        if (credentials) {
+            newOptions.credentials = credentials;
+        }
+
+        const response = await fetch(url, newOptions);
+
+        if (response.ok) {
+            return response.json();
+        }
+
+        const text = await response.text();
+
+        throw new ClientError(Client4.url, {
+            message: text || '',
+            status_code: response.status,
+            url,
+        });
+    }
 
-        const response = await request.
-            post(url).
-            send(body).
-            set(headers).
-            type('application/json').
-            accept('application/json');
+    doPost = async (url, body, headers = {}) => {
+        headers['Content-Type'] = 'application/json';
+        const options = {
+            method: 'post',
+            body: JSON.stringify(body),
+            headers,
+        };
```

Comment:
I'm seeing an issue with adding the Client4 options to API requests we send to Skype's server. I suggest a change like this to solve the issue. Also, there was an `Accept` header being used before that I think we should use to maintain consistency with how the code works. On-prem Skype could be expecting it, for example.

```suggestion
        headers['Accept'] = 'application/json';
        headers['Content-Type'] = 'application/json';
        let options = {
            method: 'post',
            body: JSON.stringify(body),
            headers,
        };
        
        if (url.includes('plugins/' + pluginID)) {
            options = Client4.getOptions(options);
        }
```